db/migrate/20210604083336_add_offender_id_to_vlo.rb

```diff
@@ -0,0 +1,7 @@
+class AddOffenderIdToVlo < ActiveRecord::Migration[6.0]
+  def change
+    change_table :victim_liaison_officers do |vlo_table|
+      vlo_table.string :nomis_offender_id, limit: 7, null: false, index: true
```

Comment:
`null: false` assumes that every VLO record has an existing Offender record – which I don't think we can assume to be true, given the current state of our database.

Reply:
@starswan can I just double check this with you? I still think it's a problem...

Surely this migration will fail in production, since `null: false` is specified but without a `default` value.

We can't run the migration rake task until the columns has been added... a bit of a chicken & egg situation?